maths/Factorial

```diff
@@ -0,0 +1,12 @@
+# Python 3 program to find  
+# factorial of given number 
+def factorial(n): 
+      
+    # single line to find factorial 
+    return 1 if (n==1 or n==0) else n * factorial(n - 1);  
```

Comment:
This code is functional for relatively small values of `n`.

Try finding factorial of `1000`